content/en/docs/tasks/manage-daemon/pods-some-nodes.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Running Pods on Only Some Nodes
+content_type: task
+weight: 30
+---
+<!-- overview -->
+
+This page shows how you can run {{<glossary_tooltip term_id="pod" text="Pods">}} on only some {{<glossary_tooltip term_id="node" text="Nodes">}} as part of a {{<glossary_tooltip term_id="daemonset" text="DaemonSet">}}
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+## Running Pods on only some Nodes
+
+Consider a scenario where we want to run a {{<glossary_tooltip term_id="daemonset" text="DaemonSet">}}, but you only need to run those daemon pods
+on nodes that have local solid state (SSD) storage. For example, the Pod might provide cache service to the
+node, and the cache is only useful when low-latency local storage is available.
+
```

Comment:
```suggestion

```
Maybe we can add something like _Follow the steps..._ and then start explaining in the form of steps.